pyvista/plotting/_property.py

```diff
@@ -398,6 +408,27 @@ def opacity(self, value: float):  # numpydoc ignore=GL08
         _check_range(value, (0, 1), 'opacity')
         self.SetOpacity(value)
 
+    @property
+    def edge_opacity(self) -> float:  # numpydoc ignore=RT01
+        """Return or set the edge opacity of this property.
+
+        Edge opacity of the mesh. A single float value that will be applied globally
+        edge opacity of the mesh and uniformly applied everywhere. Between 0 and 1.
+
+        """
+        if vtk_version_info < (9, 3):
+            return 1.0
+        else:  # pragma: no cover
+            return self.GetEdgeOpacity()
+
+    @edge_opacity.setter
+    def edge_opacity(self, value: float):  # numpydoc ignore=GL08
+        _check_range(value, (0, 1), 'edge_opacity')
+        if vtk_version_info < (9, 3):
+            pass
```

Comment:
It would be better if we could give a warning that nothing happened.  But, this gets tricky as we don't want to see a warning if it is called only from internal code.  Not sure if there is a way to get both.

Reply:
Let's try if we can do it.
```suggestion
            from pyvista.core.errors import VTKVersionError

            raise VTKVersionError(
                "`edge_opacity` cannot be used under VTK v9.3.0. Try installing VTK v9.3.0 or newer."
            )
```

Reply:
I revert this due to raising an error in document build.